skyportal/app_server.py

```diff
@@ -11,7 +11,7 @@
     GroupHandler, GroupUserHandler,
     InstrumentHandler,
     NewsFeedHandler,
-    PhotometryHandler,
+    PhotometryHandler, BulkDeletePhotometryHandler,
```

Comment:
Any reason not to incorporate this functionality into `PhotometryHandler.delete`?

Reply:
just seems simpler -- if we combine them, we'll have to check to see if the value passed in is a valid phot ID or a bulk upload ID